pincer/exceptions.py

```diff
@@ -24,8 +24,49 @@ def __init__(self, specific: str):
 
 class NoExportMethod(PincerError):
     """
-    Exception which gets raised when an `export` method is expected but not
-    found in a module.
+    Exception which gets raised when an `export` method is expected but
```

Comment:
can this be changed from \`export\` to \`\`export\`\` so that it works with the docs

Reply:
Fixed